source/data-formats/dates-and-times.txt

```diff
@@ -0,0 +1,448 @@
+      
+      print(f"UTC datetime: {utc_datetime}")
+      print(f"Local datetime: {local_datetime}")
+
+   .. output::
+      :language: shell
+   
+      UTC datetime: 2002-10-27 06:00:00
+      Local datetime: 2002-10-27 06:00:00-08:00
+
+For a canonical list of time zone strings, see the
+`Time Zone Database <https://www.iana.org/time-zones>`__ or its corresponding article
+on :wikipedia:`Wikipedia. <Tz_database>`
+
+.. important:: 
+   
+   With {+driver-short+}, you can't save ``datetime.date`` instances, because
+   there is no BSON type for dates without times. 
+   Convert all ``date`` objects to ``datetime`` objects before saving them to
+   MongoDB.
+
+Reading datetimes
+-----------------
+
+When you use {+driver-short+} to retrieve a ``datetime`` value, the driver can format
+it as naive UTC, aware UTC, or a localized value. The following sections describe how to
+retrieve each kind of value.
+
+For these sections, assume that a MongoDB collection named ``sample_collection``
+contains the following document. The value of the ``"date"`` field is a UTC ``datetime``
+value.
+
+.. code-block:: shell
+
+   {"date": datetime(2002, 10, 27, 14, 0, 0)}
+
+Naive UTC datetime
+~~~~~~~~~~~~~~~~~~
+
+By default, {+driver-short+} retrieves UTC ``datetime`` values, with no supplemental
+information. The following code example retrieves the sample document
+and prints the ``datetime`` value. The printed value is identical to
+the one in the sample document.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: python
+
+      from datetime import datetime
+
+      collection = database["sample_collection"]
+      find_result = collection.find_one()["date"]
+      print(f"datetime: {find_result}")
+      print(f"datetime.tzinfo: {find_result.tzinfo}")
+
+   .. output::
+      :language: shell
+      :visible: false
+
+      datetime: 2002-10-27 14:00:00
+      datetime.tzinfo: None
+
+Aware UTC datetime
+~~~~~~~~~~~~~~~~~~
+
+To instruct {+driver-short+} to retrieve an aware ``datetime`` value,
+create a ``CodecOptions`` object, passing ``tz_aware = True`` to the constructor.
+Then, pass the ``CodecOptions`` object to the ``get_collection()`` method.
+
+The following code example retrieves the ``datetime`` value from the sample document as
+an aware ``datetime``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: python
+
+      from pymongo import MongoClient 
+      from datetime import datetime
+      from bson.codec_options import CodecOptions
+
+      options = CodecOptions(tz_aware = True)
+      collection = database.get_collection("sample_collection", options)
+      find_result = coll.find_one()["date"]
+      
+      print(f"datetime: {find_result}")
+      print(f"datetime.tzinfo: {find_result.tzinfo}")
+
+   .. output::
+      :language: shell
+      :visible: false
+
+      datetime: 2002-10-27 14:00:00+00:00
+      datetime.tzinfo: <bson.tz_util.FixedOffset object at 0x104db2b80>
```

Comment:
I agree it's a little messy. I thought it would be good to show users what an aware UTC time looks like, so they don't assume that a datetime with tzinfo is automatically localized.

Do you have any other ideas? I know datetime has other properties that could be interesting, but I don't want to complicate this too much.